README.md

```diff
@@ -11,6 +11,10 @@ Update:
 
     $ conda env update -f environment.yml
 
+Remove:
+
+    $ conda remove --name spatial_stan --all
```

Comment:
I'm noticing that we've used short flag forms before (`-f` and not `--file`), but whatever.